src/utilities/util_main.c

```diff
@@ -84,7 +84,8 @@ main(int argc, char *argv[])
     size_t len;
     int ch, major_v, minor_v, tret, (*func)(WT_SESSION *, int, char *[]);
     char *p, *secretkey;
-    const char *cmd_config, *config, *p1, *p2, *p3, *readonly_config, *rec_config, *salvage_config;
+    const char *conn_config, *cmd_config, *p1, *p2, *p3, *readonly_config, *rec_config,
```

Comment:
`cmd_config` must be the first parameter.

Reply:
9cd762e94f4e505925286ddde5a806638b8c5359